src/components/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,78 @@
+import { Dialog, Transition } from '@headlessui/react'
+import { Fragment, useState } from 'react'
+
+export default function ConfirmationDialog({isOpen, setIsOpen, name, deleteFn}: any) {
+  function closeModal() {
+    setIsOpen(false)
+  }
+
+  function openModal() {
+    setIsOpen(true)
+  }
+
+  return (
+      <Transition appear show={isOpen} as={Fragment}>
+        <Dialog as="div" className="relative z-10" onClose={closeModal}>
+          <Transition.Child
+            as={Fragment}
+            enter="ease-out duration-300"
+            enterFrom="opacity-0"
+            enterTo="opacity-100"
+            leave="ease-in duration-200"
+            leaveFrom="opacity-100"
+            leaveTo="opacity-0"
+          >
+            <div className="fixed inset-0 bg-black/25" />
+          </Transition.Child>
+
+          <div className="fixed inset-0 overflow-y-auto">
+            <div className="flex min-h-full items-center justify-center p-4 text-center">
+              <Transition.Child
+                as={Fragment}
+                enter="ease-out duration-300"
+                enterFrom="opacity-0 scale-95"
+                enterTo="opacity-100 scale-100"
+                leave="ease-in duration-200"
+                leaveFrom="opacity-100 scale-100"
+                leaveTo="opacity-0 scale-95"
+              >
+                <Dialog.Panel className="w-full max-w-md transform overflow-hidden rounded-2xl bg-[#161f29] p-6 text-left align-middle shadow-xl transition-all border border-[#dc2625]">
```

Comment:
Let's use the tailwind color schemes instead of specific color codes and with preference for those already in use in the app for visual consistency. 
The red border: suggested the same color as used in the Settings page.
Use `rounded-lg` for consistency with the rest of the app.
The background color should match the 'theme' (dark/light). Suggesting the same colors as used in the Section component.
Please change the other places and buttons in this component to follow the same idea.

```suggestion
                <Dialog.Panel className="w-full max-w-md transform overflow-hidden rounded-lg bg-white dark:bg-gray-900 p-6 text-left align-middle shadow-xl transition-all border border-red-600">
```

Reply:
Done 🚀